pkg/cloudprovider/gce/gce.go

```diff
@@ -177,7 +178,11 @@ func newGCECloud(config io.Reader) (*GCECloud, error) {
 			projectID = cfg.Global.ProjectID
 		}
 		if cfg.Global.NetworkName != "" {
-			networkName = cfg.Global.NetworkName
+			if strings.Contains(cfg.Global.NetworkName, "/") {
```

Comment:
It should be fine. Just splitting up the universe of possible inputs in different ways.

An invalid cfg.Global.NetworkName is going to cause things to break down the line whether we treat is as a name or a full URL.